README.md

```diff
@@ -87,7 +87,7 @@ The above information is a quick path to enable running Kai quickly to see how i
 ## Roadmap and Early Builds
 
 - Kai is in it's early development phase and is NOT ready for production usage.
-- See [ROADMAP.md](/ROADMAP.md) to learn about the project's goals and milestones
+- See [docs/ROADMAP.md](docs/ROADMAP.md) to learn about the project's goals and milestones
```

Comment:
I'm unsure about this relocation of ROADMAP.md.   I lean towards thinking keeping it at root directory of project is good so its very explicit for anyone coming into the project. 

I am thinking we'd want to keep these files at root level;  README.md, ROADMAP.md, CONTRIBUTING.md